--- a/content/browser/renderer_host/navigation_controller_impl_unittest.cc
+++ b/content/browser/renderer_host/navigation_controller_impl_unittest.cc
@@ -165,6 +165,11 @@
               (bool supports_draggable_regions),
               (override));

+  MOCK_METHOD(void,
+              UpdateHeliumNoiseTokens,
+              (const blink::HeliumNoiseTokenMap& tokens),
+              (override));
+
   MOCK_METHOD(void, UpgradePrerenderUntilScriptToFullPrerender, (), (override));

   mojo::PendingAssociatedRemote<blink::mojom::PageBroadcast> GetRemote() {
--- a/content/test/test_page_broadcast.cc
+++ b/content/test/test_page_broadcast.cc
@@ -71,4 +71,7 @@
 void TestPageBroadcast::SetSupportsDraggableRegions(
     bool supports_draggable_regions) {}

+void TestPageBroadcast::UpdateHeliumNoiseTokens(
+    const blink::HeliumNoiseTokenMap& tokens) {}
+
 }  // namespace content
--- a/content/test/test_page_broadcast.h
+++ b/content/test/test_page_broadcast.h
@@ -51,6 +51,7 @@
   void UpdateColorProviders(
       const blink::ColorProviderColorMaps& color_provider_colors) override;
   void SetSupportsDraggableRegions(bool supports_draggable_regions) override;
+  void UpdateHeliumNoiseTokens(const blink::HeliumNoiseTokenMap& tokens) override;

   mojo::AssociatedReceiver<blink::mojom::PageBroadcast> receiver_;
 };
--- a/third_party/blink/renderer/core/frame/frame_test_helpers.cc
+++ b/third_party/blink/renderer/core/frame/frame_test_helpers.cc
@@ -791,7 +791,8 @@
       /*page_base_background_color=*/std::nullopt, browsing_context_group_token,
       /*color_provider_colors=*/nullptr,
       /*history_index=*/-1,
-      /*history_length=*/0));
+      /*history_length=*/0,
+      /*noise_tokens=*/blink::HeliumNoiseTokenMap()));
   // This property must be set at initialization time, it is not supported to be
   // changed afterward, and does nothing.
   web_view_->GetSettings()->SetViewportEnabled(viewport_enabled_);
@@ -835,7 +836,8 @@
       /*browsing_context_group_token=*/base::UnguessableToken::Create(),
       /*color_provider_colors=*/nullptr,
       /*history_index=*/-1,
-      /*history_length=*/0));
+      /*history_length=*/0,
+      /*noise_tokens=*/blink::HeliumNoiseTokenMap()));
 }

 int TestWebFrameClient::loads_in_progress_ = 0;
--- a/third_party/blink/renderer/core/page/page_test.cc
+++ b/third_party/blink/renderer/core/page/page_test.cc
@@ -23,7 +23,8 @@

   Page* page =
       Page::CreateOrdinary(*client, /*opener=*/nullptr, *scheduler, bcg_token,
-                           /*color_provider_colors=*/nullptr);
+                           /*color_provider_colors=*/nullptr,
+                           /*noise_tokens=*/HeliumNoiseTokenMap());

   EXPECT_EQ(page->BrowsingContextGroupToken(), bcg_token);
 }
@@ -47,7 +48,8 @@

   Page* page = Page::CreateOrdinary(*client, /*opener=*/nullptr, *scheduler,
                                     initial_bcg_token,
-                                    /*color_provider_colors=*/nullptr);
+                                    /*color_provider_colors=*/nullptr,
+                                    /*noise_tokens=*/HeliumNoiseTokenMap());

   EXPECT_EQ(page->BrowsingContextGroupToken(), initial_bcg_token);

@@ -70,7 +72,8 @@

   Page* page1 =
       Page::CreateOrdinary(*client, /*opener=*/nullptr, *scheduler, group_a,
-                           /*color_provider_colors=*/nullptr);
+                           /*color_provider_colors=*/nullptr,
+                           /*noise_tokens=*/HeliumNoiseTokenMap());

   auto pauser_for_group_a =
       std::make_unique<ScopedBrowsingContextGroupPauser>(*page1);
@@ -82,7 +85,8 @@

   Page* page2 =
       Page::CreateOrdinary(*client, /*opener=*/nullptr, *scheduler, group_b,
-                           /*color_provider_colors=*/nullptr);
+                           /*color_provider_colors=*/nullptr,
+                           /*noise_tokens=*/HeliumNoiseTokenMap());
   ASSERT_FALSE(page2->Paused());

   page2->UpdateBrowsingContextGroup(group_a);
@@ -100,7 +104,8 @@
   auto color_provider_colors = ColorProviderColorMaps::CreateDefault();

   Page* page = Page::CreateOrdinary(*client, /*opener=*/nullptr, *scheduler,
-                                    bcg_token, &color_provider_colors);
+                                    bcg_token, &color_provider_colors,
+                                    /*noise_tokens=*/HeliumNoiseTokenMap());

   const ui::ColorProvider* light_color_provider =
       page->GetColorProviderForPainting(
--- a/third_party/blink/renderer/core/testing/dummy_page_holder.cc
+++ b/third_party/blink/renderer/core/testing/dummy_page_holder.cc
@@ -103,7 +103,8 @@
   page_ = Page::CreateOrdinary(*chrome_client, /*opener=*/nullptr,
                                *agent_group_scheduler_,
                                base::UnguessableToken::Create(),
-                               /*color_provider_colors=*/nullptr);
+                               /*color_provider_colors=*/nullptr,
+                               /*noise_tokens=*/HeliumNoiseTokenMap());
   Settings& settings = page_->GetSettings();
   if (setting_overrider)
     std::move(setting_overrider).Run(settings);
--- a/third_party/blink/renderer/modules/peerconnection/webrtc_audio_renderer_test.cc
+++ b/third_party/blink/renderer/modules/peerconnection/webrtc_audio_renderer_test.cc
@@ -252,7 +252,8 @@
             /*browsing_context_group_token=*/base::UnguessableToken::Create(),
             /*color_provider_colors=*/nullptr,
             /*history_index=*/-1,
-            /*history_length=*/0)),
+            /*history_length=*/0,
+            /*noise_tokens=*/blink::HeliumNoiseTokenMap())),
         web_local_frame_(blink::WebLocalFrame::CreateMainFrame(
             web_view_,
             &web_local_frame_client_,
--- a/third_party/blink/renderer/modules/webrtc/webrtc_audio_device_impl_test.cc
+++ b/third_party/blink/renderer/modules/webrtc/webrtc_audio_device_impl_test.cc
@@ -220,7 +220,8 @@
         /*browsing_context_group_token=*/base::UnguessableToken::Create(),
         /*color_provider_colors=*/nullptr,
         /*history_index=*/-1,
-        /*history_length=*/0);
+        /*history_length=*/0,
+        /*noise_tokens=*/blink::HeliumNoiseTokenMap());

     web_local_frame_ = blink::WebLocalFrame::CreateMainFrame(
         web_view_, &web_local_frame_client_, nullptr, mojo::NullRemote(),
